service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java

```diff
@@ -952,7 +953,8 @@ public Map<String, OrganisationPolicy> changeThirdPartyStandaloneApplicant(CaseD
             .organisationName(orgName)
             .build();
 
-        var applicantCaseRole = caseData.getOutsourcingPolicy().getOrgPolicyCaseAssignedRole();
+        var applicantCaseRole = caseData.getOutsourcingPolicy() != null
+            ? caseData.getOutsourcingPolicy().getOrgPolicyCaseAssignedRole() : CaseRole.EPSMANAGING.formattedName();
```

Comment:
wonder if this would be better off parameterized? (just incase we have to add an LAMANAGING at some point)